packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/index.tsx

```diff
@@ -150,6 +158,12 @@ export default function SpanDetail(props: SpanDetailProps) {
               focusSpan={focusSpan}
             />
           )}
+        <div>
+          <a href={linkPatternsUrl} target="_blank" rel="noopener noreferrer">
+            <Icon type="info-circle-o" className="SpanDetail--docsIcon" />
+            Learn how to configure links
```

Comment:
Is this being shown within every span?

Reply:
it is, any other idea would be appreciated :)
we can also remove that part from this PR and do it in a separate one

Reply:
I think it would be too much to have this as part of every single span in a trace. We might have hundreds/thousands of them...

Reply:
ok, we'll try to find a better location for it or just remove it from this PR

Reply:
As I already wrote earlier, this information shouldn't be in the UI because the end user of the UI cannot change the configuration, only the operator can.